### DISCUSSION TITLE
new functionality

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#2242] [Q&A]
@lfcnassif  Do you remember that during the last course you taught on IPED, we mentioned the possibility of having an option to record, externally, a .txt file with all the indexed words in the media, to later use as a source of attack on encrypted files?
Will this be possible in version 4.2?

paulo

**Comment by lfcnassif** [comment]
Hi @paulobreim. What you described is possible since years ago using the Options button -> Export Indexed Word List.

If you mean exporting keywords keeping their original cases and special chars unchanged, as I explained to you privately, the challenge to export original words from files right after parsing them is to deduplicate words within the same file and across different files, otherwise you will get huge word list files. If you find an efficient solution for that without changing the index options, the current recommended approach, let me know.

**Reply by paulobreim** [reply]
I'll take a look at this.
In the case of exporting the original keywords, is there a way for iped to call an api passing each word?
If it's possible, I could make a program that receives this word and would use some way to check for duplicates.

**Reply by lfcnassif** [reply]
Yes, it is possible, but unfortunately it is not a priority for 4.2, the priorities are here:
https://github.com/sepinf-inc/IPED/projects?type=classic
